README.md

```diff
@@ -133,7 +133,7 @@ Get pyenv-win via one of the following methods:
    2. Now add the following paths to your USER PATH variable in order to access the pyenv command. Run the following in PowerShell or Windows 8/above Terminal:
       
       ```
-      [System.Environment]::SetEnvironmentVariable('path', $env:USERPROFILE + "\.pyenv\pyenv-win\bin;" + $env:USERPROFILE + "\.pyenv\pyenv-win\shims;" + [System.Environment]::GetEnvironmentVariable('path', "User"),"User")
+      [System.Environment]::SetEnvironmentVariable('path', $env:USERPROFILE + "\.pyenv\pyenv-win\bin;" + $env:USERPROFILE + "\.pyenv\pyenv-win\bin;" + [System.Environment]::GetEnvironmentVariable('path', "User"),"User")
```

Comment:
We do need to add both bin and shims folders. Why do you want to add bin folder twice and remove shims from PATH? This will not work obviously.